.github/workflows/release.yml

```diff
@@ -53,7 +53,6 @@ jobs:
       - uses: taiki-e/upload-rust-binary-action@v1
         with:
           bin: urdf-viz
-          archive: $bin-$tag-$target
```

Comment:
`$bin-$target` (e.g., `urdf-viz-x86_64-unknown-linux-gnu`) is [the default of taiki-e/upload-rust-binary-action](https://github.com/taiki-e/upload-rust-binary-action#configuration). So this PR just remove this line.